### ISSUE TITLE
ScenarioManager.load_data_from_excel can't find file on CPDaaS

### ISSUE THREAD

**Issue by victorterpstra** [IBM/dse-decision-optimization-utilities#38]
ScenarioManager.load_data_from_excel  (v0.5.4.0) in CPDaaS is trying to read from `/home/wsuser/work/`, but assets are not there.

**Comment by victorterpstra** [comment]
Patch for use in notebook cell:
```
from typing import Dict, Tuple
Inputs = Dict[str, pd.DataFrame]
Outputs = Dict[str, pd.DataFrame]
InputsOutputs = Tuple[Inputs, Outputs]
from dse_do_utils.scenariomanager import Platform
import pathlib

def load_data_from_excel(self, excel_file_name: str) -> InputsOutputs:
    """Load data from an Excel file located in the `datasets` folder of the root directory.
    Convenience method.
    If run not on WS, requires the `root_dir` property passed in the ScenarioManager constructor
    """
    if pathlib.Path(excel_file_name).suffix == '.xlsx':
        file_name = excel_file_name
    else:
        file_name = excel_file_name + '.xlsx'

    if self.platform == Platform.CPDaaS:
        # For CPDaaS only: file doesn't exist in /home/wsuser/work/. We have to get it.
        file = self.project.get_file(file_name)
        file.seek(0)
        xl = pd.ExcelFile(file)
    else:
        datasets_dir = self.get_data_directory()
        excel_file_path = os.path.join(datasets_dir, file_name)
        xl = pd.ExcelFile(excel_file_path)
        
    # Read data from Excel
    self.inputs, self.outputs = ScenarioManager.load_data_from_excel_s(xl)
    return self.inputs, self.outputs
ScenarioManager.load_data_from_excel = load_data_from_excel
```

**Comment by victorterpstra** [comment]
The fix based on project_lib is included in v0.5.4.1b
